src/Common/src/Interop/Windows/SspiCli/SecuritySafeHandles.cs

```diff
@@ -694,16 +696,19 @@ internal abstract partial class SafeDeleteContext : SafeHandle
                     fixed (byte* pinnedOutBytes = outSecBuffer.token)
                     {
                         // Fix Descriptor pointer that points to unmanaged SecurityBuffers.
-                        Interop.SspiCli.SecBuffer outUnmanagedBuffer = default;
-                        outSecurityBufferDescriptor.pBuffers = &outUnmanagedBuffer;
+                        outSecurityBufferDescriptor.pBuffers = outUnmanagedBufferPtr;
```

Comment:
Can this instead be:
```C#
outSecurityBufferDescriptor.pBuffers = &outUnmanagedBuffer[0];
```
and then you don't need the extra fixed statement above?

Reply:
is it because outUnmanagedBuffer is stackalloc and therefore willnever move? 
I get `error CS0212: You can only take the address of an unfixed expression inside of a fix
ed statement initializer` when I tried it.